src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ThreadPoolTaskScheduler.cs

```diff
@@ -47,7 +47,9 @@ protected internal override void QueueTask(Task task)
                 // Run LongRunning tasks on their own dedicated thread.
                 Thread thread = new Thread(s_longRunningThreadWork);
                 thread.IsBackground = true; // Keep this thread from blocking process shutdown
+#if !TARGET_BROWSER
```

Comment:
Please if-def out whole block to remove also `s_longRunningThreadWork` lambda

Reply:
We previously talked about changing Thread for browser to just queue a work item to the thread pool.  Is there a reason we're not actively considering that?

Reply:
No but I think that might be better done in separate PR

Reply:
I agree.  But I'd prefer we do that rather than sprinkling these ifdefs.

Reply:
> Can we hold off on this until that's fixed?

Can we revert mono related changes instead? Changes made here for mono corelib causing lot more failures in libraries build too, i have started fixing them but it would be better and cleaner if we have the path for browser.

Reply:
I'd not revert the changes but I'm fine with commenting out UnsupportedOSPlatform on problematic APIs and filling issue for that (hiding them with pragma could work too if you create an issue for each pragma)

Reply:
> Please if-def out whole block to remove also s_longRunningThreadWork lambda

If include whole block getting another warning about unused field `Unused field 's_longRunningThreadWork'`, if def the lambda declaration or maybe just leave it as is

`ThreadPoolTaskScheduler.cs(32,58): error CA1823: Unused field 's_longRunningThreadWork'`